src/auth-service/models/Maintenance.js

```
@@ -0,0 +1,248 @@
+        errors = err.keyValue;
+        message = "duplicate values provided";
+        status = httpStatus.CONFLICT;
+        Object.entries(errors).forEach(([key, value]) => {
+          return (response[key] = value);
+        });
+      } else {
+        message = "validation errors for some of the provided fields";
+        status = httpStatus.CONFLICT;
+        errors = err.errors;
+        Object.entries(errors).forEach(([key, value]) => {
+          return (response[key] = value.message);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)